packages/manager/apps/vrack-services/src/components/VrackServicesDataGridCells.tsx

```diff
@@ -34,14 +35,17 @@ export const DisplayNameCell: React.FC<DataGridCellProps<
     UpdateVrackServicesParams
   >;
   navigate?: NavigateFunction;
-}> = ({ cellData, rowData, updateVS, navigate }) => {
+  trackClick: (prop: TrackingProps) => PromiseLike<void>;
+}> = ({ cellData, rowData, updateVS, navigate, trackClick }) => {
   const displayName = cellData || rowData?.id;
   return (
     <EditableText
       disabled={!isEditable(rowData)}
       defaultValue={cellData}
-      dataTracking="vrack-services::overview::edit"
-      confirmDataTracking="vrack-services::overview::update::confirm"
+      dataTrackingPath="overview"
```

Comment:
is the "vrack-services" prefix handle directly by the trackClick function depending of the uapp ?